### PR TITLE
Refactor 05 cross chain configuration tests

### PR DESCRIPTION
**Pull request by mmvazzano**
## Describe Changes

- Modify section headings using Comment Divider
- Get deployer address from Constants.sol
- Removed the unused deployTestSampleERC20 function
- Three new tests (previously omitted) have been written and successfully passed.
- Two other tests were written (previously skipped) but still failed.
- It remains to check whether the Holpographer addresses should be identical or different in the two chains.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All Foundry tests are passing

### CODE REVIEW

File: test/foundry/deploy/05_cross-chain_configuration_tests_lq_l2.sol [hunk 1]
```diff
@@ -140,7 +138,8 @@ contract CrossChainConfiguration is Test {
   function deployTestHToken(bool isChain1) private returns (DeploymentConfig memory, bytes32, Verification memory) {
     string memory tokenName = string.concat("Holographed TestToken chain ", ((isChain1) ? "one" : "two"));
     DeploymentConfig memory deployConfig = HelperDeploymentConfig.getDeployConfigERC20(
-      bytes32(0x000000000000000000000000000000000000486f6c6f67726170684552433230), //hToken hash
+      //bytes32(0x000000000000000000000000000000000000486f6c6f67726170684552433230), //hToken hash
```
Comment (fgregoribattista) [inline test/foundry/deploy/05_cross-chain_configuration_tests_lq_l2.sol]: why this change?

**Review by fgregoribattista — APPROVED** [review]
only one thing. The rest its ok for me

**Review by djaciel — APPROVED** [review]

File: test/foundry/deploy/05_cross-chain_configuration_tests_lq_l2.sol (hunk 1 above)
Comment (mmvazzano) [reply test/foundry/deploy/05_cross-chain_configuration_tests_lq_l2.sol]: It's a bug in the code, already fixed.

**Review by sogoiii — APPROVED** [review]
👍
